### ISSUE TITLE
Detekt complains about line length... about a url... in a kdoc comment

### ISSUE THREAD

**Issue by ColtonIdle** [detekt/detekt#6209]
## Expected Behavior
Don't complain

## Observed Behavior
Complains about a url in a kdoc comment

## Your Environment
Using latest stable detekt and gradle. on macOS.

**Comment by marschwar** [comment]
Could you please provide more information such as the rule in violation, your configuration and some example code.

**Comment by atulgpt** [comment]
Hi @marschwar, I think @ColtonIdle is referring to the below code
```
val a = 9 // This is a long comment with the URL at the end https://github.com/detekt/detekt/issues/6209
```

The above code will give the violation but we can't break the URL

**Comment by cortinico** [comment]
I suppose the rule is `MaxLineLength`.
We should ignore the lines that contain `https://` or `http://` as they're generally harder to split.

**Comment by marschwar** [comment]
The reason I was asking for more information is, that `MaxLineLength` already ignores lines that end on urls.

**Comment by cortinico** [comment]
> The reason I was asking for more information is, that `MaxLineLength` already ignores lines that end on urls.

Yup you're right, I see you have an URL Regex matcher.
@ColtonIdle can you provide a snippet?

**Comment by ColtonIdle** [comment]
Let me try to repro the issue in my project and I'll come back with a snippet!

**Comment by atulgpt** [comment]
Hi @marschwar / @cortinico in the code only the URLs at the end place are ignored. @ColtonIdle in your case is the URL coming in between the comments?

**Comment by ColtonIdle** [comment]
Yes. I found the place where I had the issue. Essentially I have

```
/**
 * This is a problem I'm having which was filed [here]()https://issuetracker.google.com/issues/223217998)
 * and the problem is still occurring so I'm now creating this one off component to handle this
 * issue.
 */
```

**Comment by cortinico** [comment]
> Yes. I found the place where I had the issue. Essentially I have
> 
> ```
> /**
>  * This is a problem I'm having which was filed [here]()https://issuetracker.google.com/issues/223217998)
>  * and the problem is still occurring so I'm now creating this one off component to handle this
>  * issue.
>  */
> ```

Yup that's a valid scenario and needs some tweaking

**Comment by atulgpt** [comment]
Hi @ColtonIdle I think `[here]()https://issuetracker.google.com/issues/223217998)` is not valid markdown syntax can you try with `[here](https://issuetracker.google.com/issues/223217998)` instead and see if the issue is coming or not?

**Comment by JacobAMason** [comment]
This was addressed in https://github.com/detekt/detekt/issues/5493 and merged in https://github.com/detekt/detekt/pull/5683

**Comment by cortinico** [comment]
@ColtonIdle can you try your snippet again in 1.23.1?

**Comment by ColtonIdle** [comment]
that worked! amazing
